webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+
+from tests.support.sync import AsyncPoll
+from .. import assert_navigation_info
+
+pytestmark = pytest.mark.asyncio
+
+DOM_CONTENT_LOADED_EVENT = "browsingContext.domContentLoaded"
+
+
+async def test_unsubscribe(bidi_session, inline, top_context):
+    await bidi_session.session.subscribe(events=[DOM_CONTENT_LOADED_EVENT])
+    await bidi_session.session.unsubscribe(events=[DOM_CONTENT_LOADED_EVENT])
+
+    # Track all received browsingContext.domContentLoaded events in the events array
+    events = []
+
+    async def on_event(method, data):
+        events.append(data)
+
+    remove_listener = bidi_session.add_event_listener(
+        DOM_CONTENT_LOADED_EVENT, on_event
+    )
+
+    url = inline("<div>foo</div>")
+
+    # When navigation reaches complete state,
+    # we should have received already browsingContext.domContentLoaded event
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    assert len(events) == 0
+
+    remove_listener()
+
+
+async def test_subscribe(bidi_session, inline, new_tab, wait_for_event):
+    # Unsubscribe in case a previous tests subscribed to the event
+    await bidi_session.session.unsubscribe(events=[DOM_CONTENT_LOADED_EVENT])
+
+    await bidi_session.session.subscribe(events=[DOM_CONTENT_LOADED_EVENT])
+
+    on_entry = wait_for_event(DOM_CONTENT_LOADED_EVENT)
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(context=new_tab["context"], url=url)
+    event = await on_entry
+
+    assert_navigation_info(event, new_tab["context"], url)
+
+    await bidi_session.session.unsubscribe(events=[DOM_CONTENT_LOADED_EVENT])
+
+
+async def test_iframe(bidi_session, new_tab, test_page, test_page_same_origin_frame):
+    # Unsubscribe in case a previous tests subscribed to the event
+    await bidi_session.session.unsubscribe(events=[DOM_CONTENT_LOADED_EVENT])
+
+    events = []
+
+    async def on_event(method, data):
+        events.append(data)
+
+    remove_listener = bidi_session.add_event_listener(
+        DOM_CONTENT_LOADED_EVENT, on_event
+    )
+    await bidi_session.session.subscribe(events=[DOM_CONTENT_LOADED_EVENT])
+
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"], url=test_page_same_origin_frame
+    )
+
+    wait = AsyncPoll(
+        bidi_session, message="Didn't receive dom content loaded events for frames"
+    )
+    await wait.until(lambda _: len(events) >= 2)
+    assert len(events) == 2
```

Comment:
In our implementation, the created iFrame first is created with `about:blank`, and then is navigated to the desired URL, meaning at this point there will be 3 events: 
1. parent with `test_page_same_origin_frame`.
2. frame with `about:blank`.
3. frame with `test_page`.

Reply:
Alright, I've updated the test to make it not sensitive to these differences